js/models.js

```diff
@@ -346,13 +357,20 @@ models.service('models', ['$rootScope', '$filter', function($rootScope, $filter)
             });
         }
 
+        // flatten content and prefix
         var rtext = "";
         for (var i = 0; i < content.length; ++i) {
             rtext += content[i].text;
         }
 
+        var rprefix = "";
```

Comment:
hm actually I'm not sure about this anymore, for most messages this is wasted work